PP_APP/server/frontend/templates/frontend/success.html

```diff
@@ -0,0 +1,41 @@
+{% extends "frontend/base.html" %}
```

Comment:
Good enough for now. We'll update this page a bit when we start hooking it up to the backend. Really depends on what kind of API from Django we settle on.